src/model/mitra-unfollow.ts

```diff
@@ -0,0 +1,17 @@
+import { ActivityObject, Link } from "activitypub-objects";
+import { Activities } from "activitypub-objects/dst/activities/activity";
+import { Activity } from "./mitra-activity";
+
+export class Unfollow implements Activity {
```

Comment:
There's not such thing like an "Unfollow" activity in the standard.

`if (object.type == Activities.UNDO && object.object.type == Activities.FOLLOW)` => it's an unfollow

```suggestion
export class Undo implements Activity {
```

i think it would be great to have it already generalized. That way we can easily implement other activities very quickly in the future.